AppInspector/rules/default/tools/pipeline.json

```diff
@@ -0,0 +1,342 @@
+[
+  {
+    "name": "Pipeline Tool (AntiMalware)",
+    "id": "AI053300",
+    "description": "Pipeline Tool (AntiMalware)",
+    "tags": [
+      "Metadata.Pipeline.Tools.AntiMalware"
+    ],
+    "applies_to": ["yaml"],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "AntiMalware",
+        "type": "regex",
+        "scopes": [ "code", "comment" ],
+        "modifiers": [ "i" ],
+        "confidence": "high",
+        "_comment": ""
+      }
+    ]
+  },
+  {
+    "name": "Pipeline Tool (Bandit)",
+    "id": "AI053400",
+    "description": "A tool designed to find common security issues in Python code",
+    "tags": [
+      "Metadata.Pipeline.Tools.Bandit"
+    ],
+    "applies_to": ["yaml"],
+    "severity": "moderate",
+    "patterns": [
+      {
+        "pattern": "Bandit@1",
+        "type": "regex",
```

Comment:
Can just be a string. (I'll avoid comment spamming, but a lot of these should just be turned into strings.)

Reply:
yes -started with a copy pattern thanks will adjust